apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/helpers.php

```diff
@@ -192,3 +192,36 @@ function populate_wp_template_data() {
 }
 register_activation_hook( __FILE__, __NAMESPACE__ . '\populate_wp_template_data' );
 add_action( 'switch_theme', __NAMESPACE__ . '\populate_wp_template_data' );
+
+/**
+ * Shows a confirm prompt when the plugin is about to be deactivated on a unlaunched site.
+ *
+ * This will filter the FSE actions on the plugin manager list to add the confirm
+ * prompt on the click event of the action=deactivate.
+ *
+ * The option launch-status exists only on wpcom sites.
+ */
+add_filter(
```

Comment:
I think this filter has to be put into the entry file, `full-site-editing-plugin.php`